example/android/build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.7.20'
```

Comment:
Can you remove this change as it has nothing to do with the dialog change.

Reply:
Removed ✅